src/puppetlabs/jdbc_util/pool.clj

```diff
@@ -172,7 +183,26 @@
            {:state :starting}))
 
        (init-error [this]
-         @init-error)))))
+         @init-error)
+
+       PoolLifetime
```

Comment:
This is not your job to fix right now but I would love it if we could eventually move these protocol definitions out of this block and have reify refer to them